fennel/sources/sources.py

```diff
@@ -366,6 +365,7 @@ class DataConnector:
     lateness: Duration
     cdc: str
     starting_from: Optional[datetime] = None
+    tiers: TierSelector
```

Comment:
Will we expect most usages of this to be a single string or a list? I suspect it's a single list but what do you think? If you also agree, I think this argument should be called "tier" (otherwise `tiers='prod'`) reads weird. 

Same for `tiers` everywhere (extractors, pipelines, sources etc)